src/index.js

```diff
@@ -1,23 +1,41 @@
-import _ from 'lodash';
 import './style.css';
-import printMe from './print.js';
 
- function component() {
-   const element = document.createElement('div');
-
-  const btn = document.createElement('button');
-
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-
-  btn.innerHTML = 'Click me and check the console!';
-
-  btn.onclick = printMe;
-
-
-  element.appendChild(btn);
-
-
-   return element;
- }
-
- document.body.appendChild(component());
\ No newline at end of file
+const tasksArray = [];
```

Comment:
- [ ] **Required:** Kindly ensure that this is populated with objects that have the following or similar attributes:

- Complated,
- Description,
- Index

Which are rendered dynamically to the page.